techniques/flash/FLASH18.html

```diff
@@ -12,7 +12,7 @@ <h3>Providing a button in the Flash to stop sound</h3>
          
             <p>This example demonstrates the addition of a button within the Flash   movie to allow the user to stop sounds from playing.  A class called   SoundHandler is created which automatically starts playing an mp3 file   when the movie loads. </p>
          
-         <pre xml:space="preserve">package wcagSamples {
+         <pre>package wcagSamples {
```

Comment:
all the naked `<pre> ... </pre>` blocks in this technique could do with inner `<code> ... </code>`